docs/src/advanced/1_variants.md

```diff
@@ -0,0 +1,267 @@
+
+# Screening model variants
+
+A major goal of COBREXA is to make exploring of many model variants easy and
+fast.
+
+One main concept that can be utilized for doing that is implemented in function
+[`screen`](@ref), which takes your model, a list of model _variants_ that you
+want to explore by some specified _analysis_, and schedules the analysis of the
+model variants on all available distributed workers.
+
+In the most basic form, using the slightly simplified variant of
+[`screen`](@ref) that is called [`screen_variants`](@ref) may work as follows:
+
+```julia
+m = load_model(StandardModel, "ecoli_core_model.json")
+
+screen_variants(
+    m,    # the model for screening
+    [
+        [],    # a variant with no modifications
+        [with_set_bound("CO2t", lb = 0, ub = 0)],   # disable CO2 transport
+        [with_set_bound("O2t", lb = 0, ub = 0)],   # disable O2 transport
+    ],
+    m -> flux_balance_analysis_dict(m, Tulip.Optimizer)["BIOMASS_Ecoli_core_w_GAM"],
+)
+```
+The call specifies a model (the `m` that we have loaded) that is being tested,
+then a vector of model variants to be created and tested, and then the analysis
+that is being run on each variant -- in this case, we find an optimal steady
+state of each of the variants, and check out the biomass production rate at
```

Comment:
check the biomass

Reply:
, and only return the biomass production rate from the dictionary returned by `flux_balance_analysis_dict(m, Tulip.Optimizer)`

Reply:
too specific, I let it a bit vague so that the people are pushed to see what's actually happening there